docker-compose-releng.yml

```diff
@@ -0,0 +1,84 @@
+version: "2.1"
+
+# TODO: what links do we need?
+services:
+  releng_db:
+    image: postgres:9.6
```

Comment:
not really, we already support running postgresql via ``make develop-run-postgresql``, we should make docker setup be a reflection of that.